docs/readmes/orc8r/override_values_of_terraform_files.md

```diff
@@ -0,0 +1,62 @@
+---
+id: override_values_of_terraform_files
+title: Override values of Terraform files
+hide_title: true
+---
+# Override values of Terraform files
```

Comment:
moving this KB to github so it can be visible to open source community. FYI @sudhikan since you helped preparing this doc.

Reply:
Thanks @wallyrb; can we change `This guide described how to` to `This guide describes how to` please? :-)